src/spectro_inlets_quantification/molecule.py

```diff
@@ -180,15 +180,19 @@ def norm_spectrum(self) -> MASS_TO_FLOAT:
     def save(self, mol_dir: Optional[PATH_OR_STR] = None, file_name: Optional[str] = None) -> None:
```

Comment:
You are right in your main comment, the order here should change for consistency with the other saveables. More in main review comment.